docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/eks-helm.mdx

```diff
@@ -0,0 +1,363 @@
+---
+id: eks-helm
+title: "EKS Helm Setup"
+description: "Guide on setting up required resources with Helm"
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## Prerequisites
+
+- a Kubernetes Cluster, see [eksctl](#) or [terraform](#) guide <!-- TODO: replace with proper links -->
```

Comment:
@Langleu did you mean to leave this TODO?

Reply:
I'm wondering if there is a formatting issue here. Usually, the GitHub UI doesn't look so... unhappy.

Reply:
Ah, I don't think this needs to be a .mdx file. Building and running locally as just a .md file produces fewer errors in the browser, and I think it should correct all the noise in the GitHub UI (all the highlighted red text). 

Is there a reason you wanted this to be a .mdx file?

Reply:
Thanks, I was under the impression that I needed to switch to `mdx` due to the Tabs component
```
Docusaurus provides the <Tabs> component that you can use in Markdown thanks to MDX.
```

Switched to `md` again and resolved all remaining `TODOs`